esp/services/ws_codesign/ws_codesignService.cpp

```diff
@@ -73,20 +80,15 @@ bool Cws_codesignEx::onSign(IEspContext &context, IEspSignRequest &req, IEspSign
         return false;
     }
 
+    StringBuffer keygrip;
     if (!isGPGv1)
     {
-        StringBuffer keygrip;
         auto kgptr = strstr(output.str(), "Keygrip = ");
         if (kgptr)
             keygrip.append(40, kgptr+10);
 
         if (keygrip.length() > 0)
-        {
-            output.clear();
-            errmsg.clear();
-            cmd.clear().appendf("gpg-connect-agent \"clear_passphrase --mode=normal %s\" /bye", keygrip.str());
-            runExternalCommand(output, errmsg, cmd.str(), "");
-        }
+            clearPassphrase(keygrip.str());
```

Comment:
The indentation may not be right here.

Reply:
@shamser the line is part of the "if" statement so the identification is fine I think. Can you double check?

Reply:
Oh... yes.